src/Nethermind/Nethermind.Serialization.SszGenerator/SszType.cs

```diff
@@ -0,0 +1,164 @@
+using Microsoft.CodeAnalysis;
+
+class SszType
+{
+    static SszType()
+    {
+        BasicTypes.Add(new SszType
+        {
+            Namespace = "System",
+            Name = "Byte",
+            Kind = Kind.Basic,
+            StaticLength = sizeof(byte),
+        });
+
+        BasicTypes.Add(new SszType
+        {
+            Namespace = "System",
+            Name = "UInt16",
+            Kind = Kind.Basic,
+            StaticLength = sizeof(ushort),
+        });
+
+        BasicTypes.Add(new SszType
+        {
+            Namespace = "System",
+            Name = "Int32",
+            Kind = Kind.Basic,
+            StaticLength = sizeof(int),
+        });
+
+        BasicTypes.Add(new SszType
+        {
+            Namespace = "System",
+            Name = "UInt32",
+            Kind = Kind.Basic,
+            StaticLength = sizeof(uint),
+        });
+
+        BasicTypes.Add(new SszType
+        {
+            Namespace = "System",
+            Name = "Int64",
+            Kind = Kind.Basic,
+            StaticLength = sizeof(long),
+        });
+
+        BasicTypes.Add(new SszType
+        {
+            Namespace = "System",
+            Name = "UInt64",
+            Kind = Kind.Basic,
+            StaticLength = sizeof(ulong),
+        });
+
+        BasicTypes.Add(new SszType
+        {
+            Namespace = "Nethermind.Int256",
+            Name = "UInt256",
+            Kind = Kind.Basic,
+            StaticLength = 32,
+        });
+
+        BasicTypes.Add(new SszType
+        {
+            Namespace = "System",
+            Name = "Boolean",
+            Kind = Kind.Basic,
+            StaticLength = 1,
+        });
+
+        BasicTypes.Add(new SszType
+        {
+            Namespace = "System.Collections",
+            Name = "BitArray",
+            Kind = Kind.Basic,
+        });
+    }
```

Comment:
Just call a method with a list of `typeof(X)`, all the rest can be crated based of that?